prover/src/memory.rs

```diff
@@ -48,7 +53,7 @@ impl PromTable {
     /// # Arguments
     /// * `cs` - [`ConstraintSystem`] to add the table to
     /// * `channels` - [`Channels`] IDs for communication with other tables
-    pub fn new(cs: &mut ConstraintSystem, channels: &Channels) -> Self {
+    pub fn new(cs: &mut ConstraintSystem, _channels: &Channels) -> Self {
```

Comment:
can you use the `let _ = ...` trick you used somewhere else also here, so that `channels` keeps its name?